go-selinux/label/label_linux_test.go

```diff
@@ -33,8 +33,13 @@ func TestInit(t *testing.T) {
 		t.Log("InitLabels Disabled Failed")
 		t.FailNow()
 	}
+	if mlabel != "system_u:object_r:container_file_t:s0:c1022,c1023" {
+		t.Log("InitLabels Disabled mlabel Failed", mlabel)
+		t.FailNow()
```

Comment:
Can be `t.Fatalf("InitLabels Disabled mlabel Failed", mlabel)` (same for other cases), but that is not important.

Reply:
Fixed